pkg/GAPJulia/JuliaInterface/gap/JuliaTranspiler.gi

```diff
@@ -0,0 +1,262 @@
+#############################################################################
+##
+##  JuliaInterface package
+##
+##  Copyright 2019
+##    Thomas Breuer, RWTH Aachen University
+##    Sebastian Gutsche, Siegen University
```

Comment:
That's a bit odd. Either list just your name, or else list all three names? Or we switch to an approach like what we now use in the GAP code, and don't list names here at all.